### PR TITLE
Proposed modifications to UG (You-ify)

### PR DESCRIPTION
**Pull request by chewypiano**
Added more YOU-language in UG and minor edits. Comment if you disagree with any of the changes.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -164,7 +168,7 @@ Examples:
 
 :information_source:
  * The default value for `PROTEIN`, `FATS` and `CARBS` is 0. The default date is the day in which the food item is added.<br>
- * Values of `PROTEIN`, `FATS` and `CARBS` must be a non-negative integer.<br>
+ * Values of `PROTEIN`, `FATS` and `CARBS` must be a non-negative integer, not exceeding 1000g.<br>
```
Comment (Jh123x) [inline docs/UserGuide.md]: Its less than 1000 rather than not exceeding

**Review by Jh123x — CHANGES_REQUESTED** [review]
Mostly LGTM.
Need to adjust the not exceeding 1000 -> Less than 1000

File: docs/UserGuide.md (hunk 1 above)
Comment (chewypiano) [reply docs/UserGuide.md]: Alright sure!

**Review by Jh123x — APPROVED** [review]
LGTM

**Pull request merged**